src/Engine/BattleInstance.java

```diff
@@ -92,7 +95,13 @@ public BattleParams(Unit attacker, Unit defender, int battleRange, boolean moved
       attackerHP = attacker.getHP();
       defenseFactor = defender.model.getDefenseRatio();
       defenderHP = defender.getHP();
-      terrainDefense = battleground.getDefLevel();
+      terrainDefense = 0;
+      // Air units shouldn't get terrain defense
+      if( ChassisEnum.AIR_HIGH != defender.model.chassis && ChassisEnum.AIR_LOW != defender.model.chassis )
+      {
+        // getDefLevel returns the number of terrain stars. Since we're using %Def, we need to multiply by 10. However, we do that when we multiply by HP in calculateDamage.
```

Comment:
If we don't need to know about multiplying by 10 at this point in the code, I might just not mention it here. Too much detail at the wrong time (when it's not relevant, in particular) can be just as confusing as not enough detail.

Reply:
I put it there because I added a multiplication by 10 because I looked at what getDefLevel did.
That comment is there to keep me from doing the same dumb thing again.

Reply:
Ah, fair enough then.